### PR TITLE
Fix: Combat Issues III

### PR DESCRIPTION
**Pull request by PedroChaparro**
Fix some issues from #108: 

- [x] Send the calculated attack when a Loomie is weakened. 
- [x] Wait a few seconds between LOOMIE_WEAKENED and UPDATE_LOOMIE events.  
- [x] Attacks timeout between LOOMIE_WEAKENED and UPDATE_LOOMIE events.

### CODE REVIEW

**Comment by PedroChaparro** [comment]
Wait for @Woynert approval.

**Review by SilviaPabon — APPROVED** [review]

**Review by Woynert — APPROVED** [review]
Great!

**Pull request merged**
